__fixtures__/test-project/.vscode/tasks.json

```diff
@@ -0,0 +1,43 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "WaitForDevServer",
+      "group": "none",
+      "type": "shell",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "-NoProfile",
+          "-ExecutionPolicy", "Bypass",
+          "$port = $env:PORT; while (-not (Test-NetConnection -ComputerName localhost -Port $port)) { Start-Sleep -Seconds 1 };"
+        ]
+      },
+      "linux": {
+        "command": "bash",
+        "args": [
+          "-c",
+          "port=$PORT; while ! nc -z localhost $port; do sleep 1; done;"
```

Comment:
I get the following output from this on ubuntu:
```
 *  Executing task: bash -c 'port=$PORT; while ! nc -z localhost $port; do sleep 1; done;' 

nc: missing port number
nc: missing port number
nc: missing port number
nc: missing port number
```

This is repeated every second.